ci/settings.xml

```diff
@@ -0,0 +1,9 @@
+<settings xmlns="http://maven.apache.org/SETTINGS/1.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.0.0 http://maven.apache.org/xsd/settings-1.0.0.xsd">
```

Comment:
name of file too generic. can you pls change to deploy_oss_settings.xml or ossrh_settings.xml or ?

Reply:
sure change.